todo/repositories/task_repository.py

```
@@ -1,41 +1,58 @@
-        tasks_collection = cls.get_collection()
-        tasks_cursor = tasks_collection.find().skip((page - 1) * limit).limit(limit)
-        return [TaskModel(**task) for task in tasks_cursor]
-
-    @classmethod
-    def count(cls) -> int:
-        tasks_collection = cls.get_collection()
-        return tasks_collection.count_documents({})
-
-    @classmethod
-    def get_all(cls) -> List[TaskModel]:
+    def list(cls, page: int, limit: int, sort_by: str = "createdAt", order: str = "desc") -> List[TaskModel]:
```

Pylint (3.3.7)

[convention] Line too long (109/100)

(C0301)